src/archive.h

```diff
@@ -374,7 +374,45 @@ namespace librealsense
 
     MAP_EXTENSION(RS2_EXTENSION_DISPARITY_FRAME, librealsense::disparity_frame);
 
-    //TODO: Define Motion Frame
+    class motion_frame : public frame
+    {
+    public:
+        motion_frame() : frame()
+        {}
+    };
+
+    MAP_EXTENSION(RS2_EXTENSION_MOTION_FRAME, librealsense::motion_frame);
+
+    class pose_frame : public frame
+    {
+    public:
+        // pose frame data buffer is pose info struct
+        struct pose_info
+        {
+            float3   translation;          /**< X, Y, Z values of translation, in meters (relative to initial position)                                    */
```

Comment:
Can you add the coordinate system definition to the struct's header : i.e Right/Left Hand CS, and its correspondence to the Depth frame CS

Reply:
@oordan can we?

Reply:
tm2 coordinate system still changes, and we fix it in the viewer. once it stabilizes we can document it (this is going to be the OVR coord system, currently it's tango). it is unrelated to depth coord system as it's a different device.